ena_upload/templates/ENA_template_runs.xml

```diff
@@ -1,12 +1,20 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<?python
+import pandas as pd
+def attributetest(row, column):
+    if hasattr(row, column) and pd.notna(row[column]) and not row[column].isspace() and str(row[column]).lower() not in ['nan', 'na']:
+        return True 
+    else:
+        return False
+?>
 <RUN_SET xmlns:py="http://genshi.edgewall.org/"
     xmlns:xi="http://www.w3.org/2001/XInclude"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:noNamespaceSchemaLocation="ftp://ftp.sra.ebi.ac.uk/meta/xsd/sra_1_6/SRA.run.xsd">
     <py:for each="alias, experiment_alias in run_groups.iteritems()">
     <RUN alias="${alias}" center_name="${center}">
         <EXPERIMENT_REF refname="${experiment_alias}"/>
-         <DATA_BLOCK>
+        <DATA_BLOCK>
             <FILES>
```

Comment:
The webin submissions portal also lists the optional field of `file_unencrypted_md5`, that in the SRA.run.xsd it seems to be called `unencrypted_checksum`.

While not in the webin submissions portal, the SRA.run.xsd also lists the optional fields:
`quality_scoring_system`
`quality_encoding`
`ascii_offset`
These contain controlled vocabulary as accepted values

Reply:
sadly I don't get if statements for attributes working...